sync/tc.py

```diff
@@ -15,11 +17,65 @@
 ARTIFACTS_BASE = "https://public-artifacts.taskcluster.net/"
 TREEHERDER_BASE = "https://treeherder.mozilla.org/"
 
+SUCCESS = "completed"
+FAIL = "failed"
+EXCEPTION = "exception"
+UNSCHEDULED = "unscheduled"
+RUNNING = "running"
+PENDING = "pending"
+
 logger = log.get_logger(__name__)
 
 env = Environment()
 
 
+class TaskclusterClient(object):
+
```

Comment:
Nit: I don't think we usually have a blank line here, unless there's a docstring or some class attributes above it.